Doc/whatsnew/3.9.rst

```diff
@@ -554,6 +554,9 @@ Removed
   defining ``COUNT_ALLOCS`` macro.
   (Contributed by Victor Stinner in :issue:`39489`.)
 
+* The ``Suite`` node from :mod:`ast` has been removed due to no longer being needed.
```

Comment:
```suggestion
* The ``ast.Suite`` node class has been removed due to no longer being needed.
```

Reply:
I'm mostly confused by "node". It's not a node, but a class. Maybe just replace "node" with "node class".

Please also update the NEWS entry.

Reply:
Agreed, `ast.AST` subclasses described as `node classes` in the documentation. Will fix